order.rb

```diff
@@ -1 +1,16 @@
 # See README.md
+class Order
+  @@prices = []
+  attr_accessor :customer, :item, :price; :orders; :items; :prices
+  def initialize(customer, item, price)
+    @customer = customer
+    @item = item
+    @price = price
+    # @@items = [] 
+    # @@prices
+  end
+  def output
+    puts"#{@item} €#{@price}"
+  end
+
+end
```

Comment:
Add newline to end of file
```suggestion
end

```